### DISCUSSION TITLE
Welcome to Forget-Me-Not Discussions! 欢迎来到讨论板块！

### DISCUSSION THREAD

**Discussion by Rad586** [Rad586/Forget-Me-Not#37] [Announcements]
✏️ Share your ideas and progressions, or ask for support here! 
✏️ 欢迎在这里分享你的点子和游戏进程，或寻求支持！

**Comment by radium-prog** [comment]
非常优秀的一款整合包，体验感很好。只是，为什么新的版本许多物品无法合成
